lang/syn/src/parser/accounts/constraints.rs

```diff
@@ -264,7 +270,7 @@ pub fn parse_token(stream: ParseStream) -> ParseResult<ConstraintToken> {
                         error: parse_optional_custom_error(&stream)?,
                     },
                 )),
-                _ => return Err(ParseError::new(ident.span(), "Invalid attribute")),
+                _ => return Err(ParseError::new(ident.span(), "Invalid attribute 4")),
```

Comment:
```suggestion
                _ => return Err(ParseError::new(ident.span(), "Invalid attribute")),
```

Reply:
My bad, I used these for testing and forgot to change it back.